lumicks/pylake/fitting/parameter_trace.py

```diff
@@ -9,6 +11,20 @@ def parameter_trace(model, params, inverted_parameter, independent, dependent, *
     all other parameters fixed. This can be used to for example invert the model with respect to
     the contour length or some other parameter.
 
+    .. note::
+
+        Inverting the model with respect to a particular parameter enforces any parameter bounds
+        defined for that parameter in `params`. Results can only be trusted for values where the
+        parameter estimate does not hit these bounds. Note that `Pylake` will issue a warning
+        whenever estimates are at the bounds.
+
+        For example, when creating a worm-like chain model the parameter `contour length` is bounded
+        between a very small value (corresponding to the length of one base pair) and infinity
+        (since zero and negative values are non-physical) by default. If the contour length is then
+        estimated to be exactly at the lower bound by this function, a warning is issued. Note
+        that these bounds can be modified by changing them through the `lower_bound` and
+        `upper_bound` attributes of a `Parameter`.
+
```

Comment:
```suggestion
    .. note::

        Inverting the model with respect to a particular parameter enforces any bounds
        defined for that parameter in `params`. Results can only be trusted for values where the
        parameter estimate does not hit these bounds. Note that `Pylake` will issue a warning
        whenever this condition is met.

        For example, when creating a worm-like chain model the parameter `contour length` has a lower bound
        of the length of one base pair by default,
        since zero and negative values are non-physical. If the contour length is then
        estimated to be exactly at the lower bound by this function, a warning is issued. Note
        that these bounds can be modified by changing them through the `lower_bound` and
        `upper_bound` attributes of a `Parameter`.

```